src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -73,11 +80,16 @@ pub struct S3MetaData<S: S3BackEnd + Send + Sync + 'static> {
     pub(crate) kv_engine: Arc<KVEngineType>,
     /// Inum allocator
     inum_allocator: INumAllocator<KVEngineType>,
+    /// Mtime cache in local
+    local_mtime: HashMap<INum, SystemTime>,
```

Comment:
Why don't we combine these two structs into a single tuple?  eg : HashMap<INum,(time,size)>

Reply:
I will fix it later.